src/com/swiper.js

```diff
@@ -0,0 +1,142 @@
+/**
+ * @author Deux Huit Huit
+ *
+ * Swiper component
+ *
+ * http://idangero.us/swiper/api
+ *
+ * @uses jQuery.fn.data()
+ * @requires https://cdnjs.cloudflare.com/ajax/libs/Swiper/4.5.1/js/swiper.min.js
+ */
+
+(function ($, win, undefined) {
+
+	'use strict';
+
+	App.components.exports('swiper', function (s) {
+		var scope = $();
+
+		var sels = $.extend({
+			element: '.js-swiper',
+			ctn: '.js-swiper-ctn',
+			wrapper: '.js-swiper-wrapper',
+			slide: '.js-swiper-slide',
+			scrollbar: '.js-swiper-scrollbar',
+			navigationPrev: '.js-swiper-navigation-previous',
+			navigationNext: '.js-swiper-navigation-next',
+			pagination: '.js-swiper-pagination',
+			preventClick: '.js-swiper-prevent-click'
+		}, s);
+
+		var nestedConfigs = [
+			'navigation',
+			'scrollbar',
+			'pagination',
+			'thumbs',
+			'autoplay',
+			'keyboard'
+		];
```

Comment:
Do you think that we should add all the **_Nested Configs_** available ? ex : **mousewheel**

If you think so, there's more that we can add ;)

Thank you

Reply:
Maybe we should define which one is added by default :)

@nitriques @fhamon @emmacharp @dominic-blain @fstgerm @alexnantel88 @f-elix @pat